src/WireMock.Net/Mapping.cs

```diff
@@ -46,7 +46,10 @@ public class Mapping : IMapping
         public bool IsStartState => Scenario == null || Scenario != null && NextState != null && ExecutionConditionState == null;
 
         /// <inheritdoc cref="IMapping.IsAdminInterface" />
-        public bool IsAdminInterface => Provider is DynamicResponseProvider || Provider is DynamicAsyncResponseProvider || Provider is ProxyAsyncResponseProvider;
+        public bool IsAdminInterface => Provider is DynamicResponseProvider || Provider is DynamicAsyncResponseProvider;

```

Comment:
It is too *dangerous* to update this code because `.IsAdminInterface` is used in more places in the application.

I think the best solution is to add a property `LogRequest` as proposed earlier.

Reply:
It actually logs both, request and response. Is `LogMapping` better name for this property?

Reply:
LogMapping is fine.